rotkehlchen/chain/evm/decoding/uniswap/utils.py

```diff
@@ -222,3 +225,84 @@ def get_position_price_from_underlying(
         ) * asset_usd_price
 
     return Price(position_price)
+
+
+def decode_uniswap_like_position_create_or_exit(
+        decoded_events: list['EvmEvent'],
+        evm_inquirer: 'EvmNodeInquirer',
+        nft_manager: 'ChecksumEvmAddress',
+        counterparty: str,
+        token_symbol: str,
+        token_name: str,
+) -> list['EvmEvent']:
+    """Decode Uniswap like position create/exit events."""
+    deposit_events, withdrawal_events, receive_events, return_events = [], [], [], []
```

Comment:
can you add to the docstring what each argument is suppose to be and what value needs to hold?